client/extensions/woocommerce/app/settings/shipping/style.scss

```diff
@@ -0,0 +1,201 @@
+.shipping__header {
+	margin-bottom: 0;
+	padding-top: 8px;
+	font-size: 18px;
+}
+
+.shipping__header-description {
+	color: $gray;
+	padding-bottom: 8px;
+	font-size: 12px;
+	line-height: 18px;
+}
+
+.shipping__credit-card-description {
+	margin-bottom: 4px;
+}
+
+.shipping__address {
+	margin-bottom: 24px;
+
+	p {
+		margin-bottom: 0;
+
+		&.shipping__address-name {
+			font-weight: bold;
+		}
+	}
+}
+
+.shipping__address-notice {
+	margin-bottom: 0;
+}
+
+.shipping__zones {
+	padding: 0;
+}
+
+.shipping__zones-row,
+.shipping__packages-row {
+	display: flex;
+	flex-direction: row;
+	padding: 16px 0;
+	align-items: center;
+
+	&:not( :last-child ) {
+		border-bottom: 1px solid lighten($gray, 20%);
```

Comment:
There should be spacing around the `parenthesis`.

https://github.com/Automattic/wp-calypso/blob/de700a238a201232c8481b59b4f7feba7700e977/docs/coding-guidelines/css.md#spacing